src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -39,7 +40,46 @@ public string OutputDirectory
                 _outputDirectory = Path.IsPathRooted(value) ? value : Path.GetFullPath(value);
                 if (!Directory.Exists(_outputDirectory))
                 {
-                    Directory.CreateDirectory(_outputDirectory);
+                    try 
```

Comment:
Could you please replace all properties with `Directory.CreateDirectory` method call without checking `Directory.Exist`  and without the try/catch block?